dss-pades-pdfbox/src/main/java/eu/europa/esig/dss/pdf/pdfbox/PdfBoxDocumentReader.java

```diff
@@ -104,18 +104,24 @@ public PdfBoxDocumentReader(DSSDocument dssDocument)
 	 * 
 	 * @param dssDocument        {@link DSSDocument} to read
 	 * @param passwordProtection {@link String} a password to open a protected document
+	 * @param memoryUsageSetting {@link MemoryUsageSetting} load setting 
 	 * @throws IOException       if an exception occurs
 	 * @throws eu.europa.esig.dss.pades.exception.InvalidPasswordException if the password is not provided or
 	 *                           invalid for a protected document
 	 */
-	public PdfBoxDocumentReader(DSSDocument dssDocument, String passwordProtection, MemoryUsageSetting memoryUsageSetting)
+	public PdfBoxDocumentReader(DSSDocument dssDocument, String passwordProtection,
+			MemoryUsageSetting memoryUsageSetting)
```

Comment:
> Instead, it will not avoid the creation of the specific OpenPdf one.

Should be possible to avoid, but you may start the implementation and we will see after how we can adjust the OpenPdf  implementation